proto/internalapi/sensor/compliance_nodescan.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option go_package = "sensor";
+
+import "google/protobuf/timestamp.proto";
+import "scanner/api/v1/note.proto";
+import "scanner/api/v1/component.proto";
+
+
+import "gogoproto/gogo.proto";
+
+package sensor;
+
+// Names and data types are designed to follow proto/storage/node.proto as close as possible
+// Next tag: 7
+message FullNodeScan {
+    string                         id         = 1 [(gogoproto.moretags) = 'search:"Node ID,store" sql:"pk"'];
```

Comment:
1. I suggest to clarify that's node id to prevent confusing reader that the id may belong to some other entity, e.g. scan id.

```suggestion
    string                         node_id         = 1 [(gogoproto.moretags) = 'search:"Node ID,store" sql:"pk"'];
```

2. I'm not sure if `gogoproto.moretags` will be used for this message, but at this point they don't hurt and we can remove them later.